src/components/head-override/HeadCommon.astro

```diff
@@ -1,7 +1,16 @@
+---
+import { SITE } from '@/consts';
+const { entry } = Astro.props
+
+// [Reference] Fetching page-title from the page's frontmatter: https://starlight.astro.build/guides/overriding-components/
+const pageTitle = `${entry?.data.title || ''} 🚀 ${SITE.title}`
```

Comment:
This is the same as the title in `HeadSEO.astro` below:

https://github.com/okTurtles/shelterprotocol.net/blob/b54ecc83615a5f599859f8d6e533d07edc4124aa/src/components/head-override/HeadSEO.astro#L9

But let me know if we want to change the sentence.